package.json

```diff
@@ -12,20 +12,24 @@
   },
   "repository": {
     "type": "git",
-    "url": "https://github.com/Inist-CNRS/node-jbj-nlp.git"
+    "url": "git+https://github.com/Inist-CNRS/node-jbj-nlp.git"
```

Comment:
I don't remember why, but we always remove the `git+` part of the repository URL.
Maybe such npm packages did not install behind a proxy...